programs/test/udp_proxy.c

```diff
@@ -131,6 +151,12 @@ static struct options
     int duplicate;              /* duplicate 1 in N packets (none if 0)     */
     int delay;                  /* delay 1 packet in N (none if 0)          */
     int delay_ccs;              /* delay ChangeCipherSpec                   */
+    char* delay_cli[MAX_DELAYED_HS];  /* handshake types of messages from
+                                      * client that should be delayed.      */
+    uint8_t delay_cli_cnt;      /* Number of entries in delay_cli.          */
+    char* delay_srv[MAX_DELAYED_HS];  /* handshake types of messages from
+                                      * server that should be delayed.      */
```

Comment:
Alignment of comment.

Reply:
Addressed in https://github.com/hanno-arm/mbedtls/commit/41038108e93f1176e7e62a67059e6dd816e484ac.